HCCGo/app/js/jobHistoryCtrl.js

```diff
@@ -30,8 +30,8 @@ jobHistoryModule.service('jobService', function() {
 
   }
 
-  $scope.loadJob = function(job) {
-
+  $scope.loadJob = function(job, clone) {
+    job.clone = clone;
```

Comment:
Ah, so you are just cloning and opening it at the same time?  I need to test this usability.